### PR TITLE
IG-16104

### PR DESCRIPTION
**Pull request by talIguaz**
validate attribute names

### CODE REVIEW

File: backends/kv/writer_test.go [hunk 1]
```diff
@@ -141,6 +141,20 @@ func (suite *WriterTestSuite) TestValidateFrameInputRepeatingShardingKey() {
 	suite.Equal("column 'col1' appears more than once as a partition key", err.Error())
 }
 
+func (suite *WriterTestSuite) TestValidateFrameInputInvalidColumnName() {
```
Comment (gtopper) [inline backends/kv/writer_test.go:144]: Also add a positive test? One for name length?

File: backends/kv/writer_test.go [hunk 2]
```diff
@@ -141,6 +141,20 @@ func (suite *WriterTestSuite) TestValidateFrameInputRepeatingShardingKey() {
...
+	column, err := frames.NewLabelColumn("****T-dawg!", 5, 1)
+	suite.NoError(err)
```
Comment (gtopper) [inline backends/kv/writer_test.go:146]: `Require()` 😛

File: backends/kv/writer_test.go [hunk 3]
```diff
@@ -141,6 +141,20 @@ func (suite *WriterTestSuite) TestValidateFrameInputRepeatingShardingKey() {
...
+	columns := []frames.Column{column}
+	frame, err := frames.NewFrame(columns, nil, nil)
+	if err != nil {
+		suite.NoError(err)
```
Comment (gtopper) [inline backends/kv/writer_test.go:150]: `Require()` 😛 
No need to check `err != nil`

File: backends/kv/writer_test.go [hunk 4]
```diff
@@ -141,6 +141,20 @@ func (suite *WriterTestSuite) TestValidateFrameInputRepeatingShardingKey() {
...
+	}
+	writeRequest := frames.WriteRequest{}
+	err = validateFrameInput(frame, &writeRequest)
+	suite.Error(err)
```
Comment (gtopper) [inline backends/kv/writer_test.go:154]: `Require()` 😛

File: backends/kv/writer_test.go [hunk 5]
```diff
@@ -141,6 +141,20 @@ func (suite *WriterTestSuite) TestValidateFrameInputRepeatingShardingKey() {
...
+	suite.Equal("column '****T-dawg!' has an invalid name", err.Error())
```
Comment (gtopper) [inline backends/kv/writer_test.go:155]: `Require()` 😛

File: backends/kv/writer_test.go (hunk 1 above)
Comment (talIguaz) [reply backends/kv/writer_test.go:139]: Added a test for name length
a positive already exist in that suite

**Pull request merged**